domain/src/main/kotlin/com/krzykrucz/elesson/currentlesson/domain/attendance/Attendance.kt

```diff
@@ -0,0 +1,42 @@
+package com.krzykrucz.elesson.currentlesson.domain.attendance
+
+import arrow.core.Either
+import com.krzykrucz.elesson.currentlesson.domain.startlesson.*
+import java.time.LocalDate
+import java.time.LocalDateTime
+
+
+sealed class Attendance {
+    data class NotCompletedAttendance(val attendance: AttendanceList)
+    data class CompletedAttendance(val attendance: AttendanceList)
+}
+
+sealed class AttendanceError {
+    data class StudentNotInRegistry(val error: String = "Student is not in registry") : AttendanceError()
+    data class LessonWasNotStarted(val error: String = "Lesson was not started"): AttendanceError()
+}
+
+
+data class AttendanceList(val className: ClassName,
+                          val date: LocalDate,
+                          val lessonHourNumber: LessonHourNumber,
+                          val presentStudents: List<Student.PresentStudent> = emptyList(),
+                          val absentStudents: List<Student.AbsentStudent> = emptyList(),
+                          val lateStudents: List<Student.LateStudent> = emptyList())
```

Comment:
attendance list doesn't contain late students, late students are just present on the list

Reply:
I added it because of the event Student Noted Late, do we want to loose that information?